lib/common/api/deprecate.js

```diff
@@ -46,6 +46,26 @@ deprecate.log = (message) => {
   }
 }
 
+// Deprecate an event.
+deprecate.event = (emitter, oldName, newName, fn) => {
```

Comment:
You never call `deprecate.event` with `fn` argument. Why is it needed?

Reply:
I don't! But we can give `deprecate.event` a function to run. So, this just makes it general.
We've done this in the past, for example:
```
deprecate.event(app, 'activate-with-no-open-windows', 'activate', function (event, hasVisibleWindows) {
  if (!hasVisibleWindows) {
    return this.emit('activate-with-no-open-windows', event)
  }
})
```

Reply:
@alexeykuzmin that was how it was originally written for extensibility by kevin :)

Reply:
@codebytere Yeah, I saw it in the removed code. But I see no reasons to keep it.
We can always add it later if needed.

Reply:
Done. Removed.